command-line/command-line-interface.md

```diff
@@ -37,7 +37,7 @@ $ cd Downloads/
 
 ### Supplemental Materials
 
-- [An introduction to bash](https://guide.bash.academy/inception/).This is a complete guide to bash. It explain its installation and need.It will help you to start using bash. It is reccomended that you should go through this guide after completing the lesson.  
+- [An introduction to bash](https://guide.bash.academy/inception/). This is a complete guide to bash. It explain its installation and need. It will help you to start using bash. It is recommended that you should go through this guide after completing the lesson.  
```

Comment:
Can you change this description to:

This is a complete guide to bash. It explains its installation and basic concepts, and has been recommended by seasoned developers.